templates/committee_manager/_space_layout.html.twig

```diff
@@ -25,16 +22,55 @@
                     <a href="{{ path('app_committee_manager_edit', {slug: committee.slug }) }}">Modifier mon comité</a>
                 </li>
 
-                <li class="{{ is_active_route(app.request, ['app_committee_manager_list_members', 'app_committee_demote_host', 'app_committee_promote_host']) ? 'active' : '' }}">
-                    <a href="{{ path('app_committee_manager_list_members', {slug: committee.slug}) }}">Mes adhérents</a>
-                </li>
-
-                {% if is_canary_enabled() %}
-                    <li class="{{ is_active_route(app.request, 'app_message_committee_*') ? 'active' : '' }}">
-                        <a href="{{ path('app_message_committee_list', {committee_slug: committee.slug}) }}">Mes messages</a>
+                {% if committee.isApproved() %}
```

Comment:
To be consistent with the others calls in the project, you can just write `committee.isApproved` without parenthesis

Like line 68

Reply:
done